RedHatInsights/registration-assistant
# chore(RHINENG-6816, RHINENG-8051): PF5 migration

https://issues.redhat.com/browse/RHINENG-6816

To test:
1. make sure npm run start and npm run build scripts work
2. compare PF4 and PF5 versions behaviour and appearences
